cli/src/vine_cli.rs

```diff
@@ -84,8 +84,19 @@ impl CompileArgs {
 
 fn std_path() -> PathBuf {
   let mut path = PathBuf::new();
-  path.push(env!("CARGO_MANIFEST_DIR"));
-  path.push("../vine/std/std.vi");
+  let compile_time_std_path = option_env!("VINE_STD_PATH");
+  let runtime_std_path = env::var("VINE_STD_PATH").ok();
+
+  match runtime_std_path.or(compile_time_std_path.map(String::from)) {
```

Comment:
very minor, but this avoids an allocation :)
```suggestion
  match runtime_std_path.as_ref().or(compile_time_std_path) {
```

Reply:
Good call!  Adjusted.